packages/react/src/components/TableOfContents/TableOfContents.js

```diff
@@ -124,7 +124,7 @@ const TableOfContents = ({
       .filter((elem, index, arr) =>
         elem.height === null
           ? arr[index - 1].position < arr[index - 1].height
-          : elem.position > -elem.height
+          : elem.position - 50 > -elem.height
```

Comment:
Just wondered where the `50` number came from, but we can go forward as long as you see this fixes the problem (and I believe so).

Reply:
That's close to the size of our navigation bar, so it would help in cases when the nav is shown, so if the component is bellow it, the next version can be considered

Reply:
I wonder if it would be clearer to maybe create a `const navBarHeight = 50` and use that so that it's more obvious?

Reply:
Yeah I think `const navBarHeight = 50` helps - Thanks @RaphaelAmadeu for your clarification and @annawen1 for your suggestion!